src/Microsoft.AspNetCore.JsonPatch/project.json

```diff
@@ -1,5 +1,11 @@
 {
   "version": "1.0.0-*",
+  "description": "ASP.NET Core support for JSON PATCH.",
```

Comment:
One interesting point here, there isn't really anything ASP.NET about this library... it only depends on JSON.NET